en/micro-integrator/docs/references/connectors/as400-pcml-connector/as400-pcml-connector-configuration.md

```diff
@@ -0,0 +1,65 @@
+    
+3. You will received the reply email with the client credentials to the your email account (It will take some hours).
+
+4. Navigate in to the [IBM i Access - Client Solutions](https://www.ibm.com/support/pages/ibm-i-access-client-solutions) page.
+
+5. Download the **IBM i Access Client Solutions**.
+
+    <img src="../../../../assets/img/connectors/as400-ibm-client-access.png" title="IBM i Access Client Solutions" width="600" alt="IBM i Access Client Solutions"/>
+
+6. Extract the downloaded zip file and install it.
+
+7. Start the IBM I access client solution.
+   
+    <img src="../../../../assets/img/connectors/as400-ibm-client-access-homepage.png" title="IBM i Access Client Solutions start" width="600" alt="IBM i Access Client Solutions start"/>
+
+8. Click **System Configurations** section under **Management** section. 
+
+    <img src="../../../../assets/img/connectors/as400-system-configuration.png" title="IBM i Access system-configuration" width="500" alt="IBM i Access system-configuration"/>
+
+9. You will prompt following window and click **New** button.
+  
+    <img src="../../../../assets/img/connectors/as400-ibm-add-new-system.png" title="IBM i Access add new system" width="600" alt="IBM i Access add new system"/>
+
+10. Add required details. Add **System name** and **Description** under General tab. In this sample we used `System name` as `pub400.com`. 
+
+    <img src="../../../../assets/img/connectors/as400-ibm-add-systemname.png" title="IBM i Access add new system name" width="600" alt="IBM i Access add new system name"/>
+
+11. Add **Default username** and **port** number under **Connection** tab. In this sample we need to use `port` number as `23`. And click **OK**.
+
+    <img src="../../../../assets/img/connectors/as400-ibm-port-and-username.png" title="IBM i Access add port number" width="600" alt="IBM i Access add port number"/>
+
+12. After setting up the configuration we can use the **5250 Emulator**. Click **5250 Emulator** link under **General** section.
+
+    <img src="../../../../assets/img/connectors/as400-ibm-5250-emulator.png" title="IBM 5250-emulator" width="600" alt="IBM 5250-emulator"/>
+
+13. When you click this emulator you need to add the username and password you got in the step 3. Then you will be see the following window.   
+
+    <img src="../../../../assets/img/connectors/as400-pub400-server-window.png" title="PUB400 server command line view" width="600" alt="PUB400 server command line view"/>
+ 
+14. You need to add the same user name and password you got in the step 3.
+
+15. Now you can see the following **IBM i Main Menu**.
+
+    <img src="../../../../assets/img/connectors/as400-ibm-i-main-menu.png" title="IBM i Main Menu" width="600" alt="IBM i Main Menu"/>
+
+For more information about these operations, please refer to the [AS400 PCML connector reference guide](as400 pcml-connector-reference.md).
```

Comment:
We do not mention above information in reference guide (Pls correct me if wrong). So we do not need this line. 
What we need to tell is username,password and information x will be required to configure the connector in the following sections.

Reply:
This information included in reference guide documentation.